### ISSUE TITLE
Cannot compile from source on R 4.1.x cannot find make

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/rig#56]
Our rtools patch is apparently not good enough.

**Comment by gaborcsardi** [comment]
Hmmm, this might be a temporary problem if we don't start a new shell? We'll keep this open until I'll know for sure....

**Comment by galenholt** [comment]
I think I'm hitting this issue- when using R 4.02, if I try to install a package (e.g. xfun) that needs Rtools, I get 
`Warning in system(paste(cmd, "shlib-clean")) : 'make' not found
Warning in system(cmd) : 'make' not found
ERROR: compilation failed for package 'xfun'`

I tried running `rig system update-rtools40`, and get the error `[ERROR] The system cannot find the path specified. (os error 3)`.

If it helps, I was already using R 4.2.1 with Rtools42 before I installed rig. Maybe the paths are getting mixed up between the rig-managed install of 4.02. and the original 4.2.1?

Thanks
